### PR TITLE
Modals

### PR DESCRIPTION
**Pull request by sambehrens**
this requires a `yarn install-all` once merged

### CODE REVIEW

File: client/src/assets/stylesheets/ui/buttons.scss [hunk 1]
```diff
@@ -99,7 +112,7 @@
 
   &:focus {
     outline: 0;
-    box-shadow: 0 0 0 3pt $blue-outline;
+    // box-shadow: 0 0 0 3pt $blue-outline;
```
Comment (karlmolina) [inline client/src/assets/stylesheets/ui/buttons.scss]: whats this

File: client/src/components/pages/dashboard/Dashboard.js [hunk 2]
```diff
@@ -9,6 +9,10 @@ const propTypes = {
 };
 
 export class Dashboard extends Component {
+    constructor() {
```
Comment (karlmolina) [inline client/src/components/pages/dashboard/Dashboard.js]: whats this

**Review by karlmolina — CHANGES_REQUESTED** [review]

File: client/src/components/pages/dashboard/Dashboard.js (hunk 2 above)
Comment (sambehrens) [reply client/src/components/pages/dashboard/Dashboard.js]: leftover code from testing

File: client/src/assets/stylesheets/ui/buttons.scss (hunk 1 above)
Comment (sambehrens) [reply client/src/assets/stylesheets/ui/buttons.scss]: forgot to delete it

**Review by karlmolina — APPROVED** [review]

**Pull request merged**